libs/measurement-history/src/lib/measurement-history.tsx

```diff
@@ -0,0 +1,140 @@
+    url: '/Measurements',
+    params: {
+      dateFrom: dateStatus.from?.format('YYYY-MM-DD HH:mm:ss').replace(' ', 'T'),
+      dateTo: dateStatus.to?.format('YYYY-MM-DD HH:mm:ss').replace(' ', 'T'),
+    },
+  });
+
+  useEffect(() => {
+    if (dateStatus.from == null || dateStatus.to == null) return;
+
+    measurementRageTrigger().then();
+  }, [dateStatus.from?.unix(), dateStatus.to?.unix()]);
+
+  return (
+    <>
+      <div className={'flex gap-4 lg:gap-8'}>
+        <h1 className={'text-2xl font-bold'}>History</h1>
+        <Divider type={'vertical'} className={'h-10 max-md:hidden'} />
+        <div className={'w-full bg-inherit z-10'}>
+          <div className={'flex items-center gap-2'}>
+            <FilterFilled />
+            <DatePicker.RangePicker
+              size={'large'}
+              use12Hours={false}
+              mode={['date', 'date']}
+              value={[dateStatus.from, dateStatus.to]}
+              onCalendarChange={val => val != null && setDateStatus({ from: val[0], to: val[1] })}
+            />
+          </div>
+        </div>
+      </div>
+      <Divider />
+      <div className={'grid gap-2 max-w-full'}>
+        <CardElement className={'h-fit xl:col-span-2'}>
+          <PieChartDiagramsCard
+            diagramData={
+              measurementRange && boundaries
+                ? {
+                    temperatureData: calculatePieChartData(
+                      measurementRange.map(m => m.temperature),
+                      boundaries.temperatureBoundaryMin,
+                      boundaries.temperatureBoundaryMax
+                    ),
+                    humidityData: calculatePieChartData(
+                      measurementRange.map(m => m.humidity),
+                      boundaries.humidityBoundaryMin,
+                      boundaries.humidityBoundaryMax
+                    ),
+                    co2Data: calculatePieChartData(
+                      measurementRange.map(m => m.co2),
+                      boundaries.cO2BoundaryMin,
+                      boundaries.cO2BoundaryMax
+                    ),
+                  }
+                : undefined
+            }
+          />
+        </CardElement>
+
+        <CardElement className={'flex max-xl:h-80 justify-center max-w-full'}>
+          <div className={'w-64 sm:w-72 md:w-96 lg:w-[40rem] xl:w-[28rem] 2xl:w-[32rem] flex justify-center'}>
+            <LineChartSummary
+              data={measurementRange?.map<LineChartSummaryData>(d => ({
+                date: dayjs(d.dateTime).format(DisplayDateFormat),
+                temperature: d.temperature,
+                humidity: d.humidity,
+                co2: d.co2,
+              }))}
+            />
+          </div>
+        </CardElement>
+
+        <CardElement className={'flex justify-center'}>
+          <div className={'w-64 sm:w-80 md:w-[30rem] lg:w-[40rem] xl:w-[28rem] 2xl:w-[40rem]'}>
```

Comment:
Could you avoid arbitrary values? it's usually good practice
<img width="783" alt="image" src="https://github.com/apeTrackunit/lizards-inc-fe/assets/78472079/30c4ea07-7a77-459a-9668-6a984fa5b02d">